oxidation/client/package.json

```diff
@@ -79,7 +79,7 @@
         "wasm-pack": "^0.10.3"
     },
     "engines": {
-        "node": ">=16.13.0 <19.0.0"
+        "node": ">=16.13.0 <18.13.0"
```

Comment:
Must add a comment to explain why this is needed !
(and ideally when upgrade will be possible)

Reply:
I can't add a comment in json :(